test/server.jl

```diff
@@ -0,0 +1,24 @@
+# find a good port
+conn = listenany(2228)
+close(conn[2])
+port = conn[1]
+
+@async lintserver(port)
+sleep(1) #let sever start
```

Comment:
Is there a point to running lintserver with `@async` if we're just going to wait for it to start?